src/Compilers/CSharp/Portable/Parser/SlidingTextWindow.cs

```diff
@@ -668,6 +668,10 @@ public string GetText(int position, int length, bool intern)
                     {
                         return " ";
                     }
+                    if (_characterWindow[offset] == '\n')
```

Comment:
In the Roslyn codebase it's custom to leave an empty line after a closing curly if it's not the last statement of the block. So if you could add an empty line before the if.